crates/boundless-market/src/contracts/hit_points.rs

```diff
@@ -50,7 +46,7 @@ where
     }
 
     /// Returns the underlying IHitPointsInstance.
-    pub fn instance(&self) -> &IHitPointsInstance<T, P, Ethereum> {
+    pub fn instance(&self) -> &IHitPointsInstance<(), P, Ethereum> {
```

Comment:
I was so confused by this until I finally found alloy's private-but-not-really traits. Very weird, but I think I see they dilemma they are trying to work around 🤷 
https://github.com/alloy-rs/alloy/blob/main/crates/contract/src/lib.rs#L39-L48